packages/client/src/components/interface/Navigation.tsx

```diff
@@ -67,6 +69,9 @@ import { IAdvancedSearchParamState } from '@client/search/advancedSearch/reducer
 import { omit } from 'lodash'
 import { getAdvancedSearchParamsState } from '@client/search/advancedSearch/advancedSearchSelectors'
 import { ADVANCED_SEARCH_RESULT } from '@client/navigation/routes'
+import { Text } from '@opencrvs/components'
+import * as Icons from 'react-feather'
+import { ChartActivity } from '@opencrvs/components/lib/Icon/custom-icons'
```

Comment:
We should rather use the `<Icon>` -component, instead of importing the icons directly. 

```
<Icons.Award width={20} height={20} />
```
-->
```
<Icon name="Award" size="medium" />
```